jsk_perception/node_scripts/people_pose_estimation_2d.py

```diff
@@ -127,9 +127,9 @@ def subscribe(self):
                 '~input', Image, queue_size=1, buff_size=2**24)
             sub_depth = message_filters.Subscriber(
                 '~input/depth', Image, queue_size=1, buff_size=2**24)
-            sub_info = message_filters.Subscriber(
-                '~input/info', CameraInfo, queue_size=1, buff_size=2**24)
-            self.subs = [sub_img, sub_depth, sub_info]
+            self.sub_info = rospy.Subscriber(
+                '~input/info', CameraInfo, self._cb_cam_info)
+            self.subs = [sub_img, sub_depth]
```

Comment:
since this is workaround and if upstream problem was solved, we can use `message_filters` so, please un-comment old implementation, instead of removing it and add more information and link to upstream issue report.

c.f. https://github.com/jsk-ros-pkg/jsk_recognition/issues/2165#issuecomment-334919526

Reply:
@k-okada True. Added comment and option `~sync_camera_info` so we can switch on/off this feature.